src/index.d.ts

```diff
@@ -465,6 +465,39 @@ declare namespace SDKStandardComponents {
     }
 
     class MojaloopRequests extends BaseRequests {
+        /**
+         * Executes a GET /parties request for the specified identifier type and identifier
+         *
+         * @returns {object} - JSON response body if one was received
```

Comment:
nit: I think this should be 
```
@returns {Promise<GenericRequestResponse | MojaloopRequestResponse>} ...
```

Although maybe it's better to leave off the return type, as it's already in the function signature.

Reply:
copy paste error :-D fixed